packages/server/src/app/services/admin/approval-queue.service.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable, Logger } from '@nestjs/common';
+import { ApprovalQueue } from '@dragonfish/models/approval-queue';
+import { PaginateResult } from 'mongoose';
+
+import { ContentKind, ContentModel } from '@dragonfish/models/content';
+import { ApprovalQueueStore } from '../../db/approval-queue/approval-queue.store';
+import { ContentStore } from '../../db/content';
+import { IApprovalQueue } from '../../shared/admin';
+
+@Injectable()
+export class ApprovalQueueService implements IApprovalQueue {
+    private readonly logger: Logger = new Logger(ApprovalQueueService.name);
+
+    constructor(private approvalQueueService: ApprovalQueueStore, private contentService: ContentStore) {}
```

Comment:
`approvalQueueService` should be renamed to `approvalQueueStore`. Ditto `contentService`.

oh god how many of these have I missed until now